lib/desired-caps.js

```diff
@@ -33,6 +33,9 @@ let desiredCapConstraints = _.defaults({
   tapWithShortPressDuration: {
     isNumber: true
   },
+  scaleFactor: {
+    isString: true
```

Comment:
Ok. It is early enough in the flow to kill the session creation when this fails. Thanks.